linstor/linstor-monitord.c

```diff
@@ -0,0 +1,402 @@
+/*
+ * Copyright (C) 2020  Vates SAS - ronan.abhamon@vates.fr
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include <ctype.h>
+#include <errno.h>
+#include <inttypes.h>
+#include <poll.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/inotify.h>
+#include <sys/ioctl.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+#include <syslog.h>
+#include <time.h>
+#include <unistd.h>
+
+// TODO: Handle new hosts.
+// TODO: https://github.com/xcp-ng/xcp/issues/421
+
+// =============================================================================
+
+#define POOL_CONF_DIR "/etc/xensource"
+#define POOL_CONF_FILE "pool.conf"
+#define POOL_CONF_ABS_FILE POOL_CONF_DIR "/" POOL_CONF_FILE
+
+// In milliseconds.
+#define POLL_TIMEOUT 2000
+
+// -----------------------------------------------------------------------------
+
+static inline void normalizeTime (struct timespec *spec) {
+  while (spec->tv_nsec >= 1000000000) {
```

Comment:
Why not put `1000000000` in a variable?
It's used 3 times and it'd be easier to read.

Reply:
Well I don't know if it would be easier to read, but if you have an explicit variable name, don't hesitate. :p